spec/features/course/assessment_condition_management_spec.rb

```diff
@@ -63,8 +69,10 @@
         expect do
           find_link(nil, href: condition_delete_path).click
         end.to change { assessment.conditions.count }.by(-1)
+        title = I18n.t('activerecord.attributes.course/condition/assessment/title.complete',
+                       assessment_title: other_assessment.title)
```

Comment:
For each for the `I18n.t`, the interpolated parts (in this case, `assessment_title`) are not checked. The tests will still pass when they are removed. Would it be better to remove them if they are redundant?